app/__init__.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+from app.config import Config
+from flask import Flask
+from flask_talisman import Talisman
+from flask_pymongo import PyMongo
+
+
+# reusable extension for PyMongo
+mongo = PyMongo()
+
+
+# create the entire instance of the app
+def create_app(config_class=Config):
+    # app initialization + configuration
+    app = Flask(__name__)
+    app.config.from_object(Config)

```

Comment:
Shouldnt you pass in the argument `config_class ` here ?

Reply:
I'm not sure I understand? Isn't that what I've got on line 13?
**def create_app(config_class=Config):**

Reply:
You pass in `config_class` as argument to the `create_app` function and used `Config` as the default argument on line 15. That argument is unused. Where do you intend to use the argument.